glean-core/ffi/glean.h

```diff
@@ -400,8 +400,7 @@ void glean_timing_distribution_accumulate_samples(uint64_t glean_handle,
 
 void glean_timing_distribution_cancel(uint64_t metric_id, TimerId timer_id);
 
-TimerId glean_timing_distribution_set_start(uint64_t metric_id,
-                                            uint64_t start_time);
+TimerId glean_timing_distribution_set_start(uint64_t metric_id, uint64_t start_time);
```

Comment:
Wow, was this change introduced automatically by our tools? Seems odd

Reply:
By cbindgen.  I think what happened actually is that I edited by hand, and then when it was regenerated the formatting changed.